<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>960</width>
      <height>370</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_ppp">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_fff">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_kkk">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_eee">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>960</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-color: rgb(53, 100, 173);
	font: 24pt &quot;휴먼매직체&quot;;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>8</number>
    </property>
    <property name="text">
     <string>포켓몬을 골라주세요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>960</width>
      <height>370</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_ppp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 100);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fff">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 100);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_kkk">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 100);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_eee">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover{
	background-color: rgba(255, 255, 255, 100);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
